src/reader/widgets/ReaderWidget.vue

```diff
@@ -55,8 +74,19 @@
       Paginator,
       Reader,
       ImageViewer,
+      ImageViewerToolbar,
     },
     scaifeConfig: {},
+    data() {
+      return {
+        showImage: 'both', // text | image | both
```

Comment:
not a blocker (and wasn't for https://github.com/scaife-viewer/explorehomer/pull/42/files#diff-7342f14685f94a5222b1cd6c709fb90cR100), but I think as part of cleaning up / extracting we might want named constants for the toggle values, since they are shared across components and could go out of sync.

We could have `IMAGE_VIEWER_STATES` and `IMAGE_VIEWER_STATE_BOTH`, etc (mimicking what we might do with ChoiceFields in Django, for example).

Have captured within ![](https://github.trello.services/images/mini-trello-icon.png) [Modularity of readers / reader components](https://trello.com/c/aDn12rrP)

Reply:
Done